src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårLovverkInformasjonForBarn.kt

```diff
@@ -19,8 +23,8 @@ data class VilkårLovverkInformasjonForBarn(
         this.periodeTomBarnetsAlderLov2021 = fødselsdato.plusYears(2)
         this.periodeFomBarnetsAlderLov2024 = fødselsdato.plusMonths(13)
         this.periodeTomBarnetsAlderLov2024 = fødselsdato.plusMonths(19)
-        val erTruffetAvLovverk2021 = periodeFomBarnetsAlderLov2021.isBefore(DATO_LOVENDRING_2024)
-        val erTruffetAvLovverk2024 = periodeTomBarnetsAlderLov2024.erSammeEllerEtter(DATO_LOVENDRING_2024)
+        val erTruffetAvLovverk2021 = periodeFomForAdoptertBarn?.isBefore(DATO_LOVENDRING_2024.toYearMonth()) ?: periodeFomBarnetsAlderLov2021.isBefore(DATO_LOVENDRING_2024)
+        val erTruffetAvLovverk2024 = periodeTomForAdoptertBarn?.isAfter(DATO_LOVENDRING_2024.toYearMonth().minusMonths(1)) ?: periodeTomBarnetsAlderLov2024.erSammeEllerEtter(DATO_LOVENDRING_2024)
```

Comment:
La til minusMonths(1) her for å få samme effekt som .sammeEllerEtter() bare for YearMonth. Gjerne gi en lyd hvis dette virker feil:)

Reply:
Jeg synes det er lettere å lese hvis man konverterer `periodeTomForAdoptertBarn` til `LocalDate` først, og så bruker `.erSammeEllerEtter`